blackbox/docs/config/secure.rst

```diff
@@ -0,0 +1,84 @@
+.. highlight:: sh
+
+.. _conf-secure-settings:
+
+===============
+Secure Settings
+===============
+
+Some settings require additional protection. For example: passwords, access 
+keys, and so on. When handling this type of data, it is typically best practice 
+not to rely on file system permissions, and not to store it on disk using plain 
+text.
+
+For sensitive configuration data, we provide a keystore_ along with the 
+``crate-keystore`` tool to manage the data in the keystore_.
+
+.. NOTE::
+
+    All commands here should be run as the user which will run CrateDB.
+
+.. NOTE::
+
+    Changes to the keystore only take effect after restarting CrateDB.
+
+.. NOTE::
+
+    The entries stored in the CrateDB keystore will be encrypted using the PBE_
```

Comment:
I'd not document the algorithm as this is implementation detail and could be changed. why not go with the previous phrase about obfuscation?